python/setup.cfg

```diff
@@ -54,4 +54,4 @@ polars =
 	%(scikit-learn)s
 
 [bdist_wheel]
-py-limited-api=cp39
\ No newline at end of file
+py_limited_api=cp39
```

Comment:
Could you add a comment explaining where this comes from? I've found this issue
- https://github.com/pypa/wheel/issues/583

and with a little searching I've done haven't found documentation for this either, or why it's underscore instead of dashes.

Reply:
I noticed the CI was complaining about dashes, so I replaced with underscores.